src/main/java/org/postgresql/util/PGobject.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Taken from Postgres JDBC Driver 9.4 (BSD license terms)
+ * 
+ * License : https://jdbc.postgresql.org/about/license.html
```

Comment:
The best thing is to fix PostGis to include the required classes. Having pgjdbc-ng depend on (directly / indirectly) or include org.postgres classes isn't going to fly.

postgis.jar can always depend on the official driver, if they won't refactor / update their code, but has that path been tried yet ?